app/javascript/lib/study-overview.js

```diff
@@ -179,36 +167,47 @@ function renderScatterPlot(target, rawPlot) {
       $(`#${plotId}`).data('camera', newCamera)
     }
   })
-
-  window.SCP.scatterPlotLayout = layout
 }
 
-/** Fetch and draw scatter plot for default Explore tab view */
-async function drawScatterPlot() {
-  const spinnerTarget = $('#plots')[0]
+/** draws scatter plot */
+async function drawScatterPlot(accession, cluster, plotIndex) {
+  const plotId = `cluster-plot-${plotIndex}`
+  const legendId = `cluster-legend-${plotIndex}`
+
+  let plotClass = '' // For only 1 plot (study without spatial data)
+  if (window.SCP.numPlots > 1) {
+    plotClass = ' plot-left'
+    if (plotIndex !== 0) {
+      plotClass = ' plot-right'
+    }
+  }
+
+  $('#plots .panel-body').append(`
+    <div class="row${plotClass}">
+      <div id="${plotId}"></div>
+      <div id="${legendId}"></div>
+    </div>`)
+
+  const plotJqDom = $(`#${plotId}`)
```

Comment:
I think the convention for delineating that a variable holds a jquery object is prefixing the variable name with $.  So $plotElement would be a more standard name for this variable

Reply:
Good point, done in 555d1f9.